frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -648,6 +648,38 @@ describe("scenarios > question > filter", () => {
     popover().contains(/case/i);
   });
 
+  it("should enable highlighting suggestions with keyboard up and down arrows (metabase#16210)", () => {
+    openReviewsTable({ mode: "notebook" });
+    cy.findByText("Filter").click();
+    cy.findByText("Custom Expression").click();
+
+    cy.get("[contenteditable='true']")
+      .click()
+      .type("c");
+
+    // First highlighted selection is Created At
+    // "C" is wrapped in a <span>
+    // so we must look for string "reated At"
+    cy.findByText("reated At")
+      .closest("li")
+      .should("have.css", "background-color")
+      .and("eq", "rgb(80, 158, 227)");
```

Comment:
I believe this color is one of the choices in `frontend/src/metabase/lib/colors.js`. I'm not sure how to avoid hardcoding the RGB values there. Any idea @nemanjaglumac?

Reply:
Cypress has access to the whole app, so it should be easy to import colors and use variable here.

Reply:
```javascript
import d3 from "d3";
import { color } from "metabase/lib/colors";

// use d3's `rgb()` function to convert HEX colors from our internal lib 
d3.rgb(color("brand")); // "brand" is just an example - choose whichever color name you need

Reply:
There is an example of this pattern in `frontend/src/metabase/visualizations/components/LeafletGridHeatMap.jsx`

Reply:
@nemanjaglumac One more thing to improve when updating webpack, perhaps.

I've declared a local const.
Worse that can happen is test breaking if/when we change this specific color in the lib.

Possible to tinker more and more, not looking like the best use of time but if you see otherwise, please let me know.

Reply:
I've reverted the logic and now it checks against a transparent background.

It is agnostic as to what color the highlight is in.

Should last longer than picking a specific color in any way: will only break if we decide to add background colors to all items…unlikely.